src/main/resources/db/migration/V7__InsertRoles.sql

```diff
@@ -0,0 +1,8 @@
+INSERT INTO `Role`
+(`roleId`, `name`)
+VALUES
+    (0, 'User'),
+    (1, 'HR'),
+    (2, 'Sales'),
+    (3, 'Management')
```

Comment:
Should this be 0 indexed or would it not be better to count from 1 for an id?

Reply:
0 is a user with no permissions